src/main/java/io/neonbee/data/DataVerticle.java

```diff
@@ -75,11 +75,206 @@
 
     @SuppressWarnings("UnnecessaryLambda") // overridden in DummyVerticleHelper, as getNamespace is final
     private final Supplier<String> namespaceSupplier =
-            () -> Optional.ofNullable(this.getClass().getAnnotation(NeonBeeDeployable.class))
```

Comment:
We decided to not staticially import methods that do not describe what they create, e.g.:

Map.of
List.of
Optional.of

Etc. otherwise there could be ambiguities